api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -226,6 +243,70 @@ public DomainValuePair apply(DataSetValues dataSetValue) {
     return ResponseEntity.ok(previewQueryResponse);
   }
 
+  @Override
+  public ResponseEntity<EmptyResponse> exportToNotebook(String workspaceNamespace, String workspaceId, DataSetExportRequest dataSetExportRequest) {
+    workspaceService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(workspaceNamespace, workspaceId, WorkspaceAccessLevel.WRITER);
+    DataSetQueryList queryList = generateQuery(workspaceNamespace, workspaceId, dataSetExportRequest.getDataSetRequest()).getBody();
+    String queriesAsStrings = "import pandas\n\n" + queryList.getQueryList().stream()
+        .map(query -> convertQueryToString(query, dataSetExportRequest.getDataSetRequest().getName())).collect(Collectors.joining("\n\n"));
+    JSONObject notebookFile = new JSONObject()
+        .put("cells", new JSONArray()
+            .put(new JSONObject()
+                .put("cell_type", "code")
+                .put("metadata", new JSONObject())
+                .put("execution_count", JSONObject.NULL)
+                .put("outputs", new JSONArray())
+                .put("source", new JSONArray()
+                  .put(queriesAsStrings))))
+        .put("metadata", new JSONObject())
+        .put("nbformat", 4)
```

Comment:
Nit: It will be helpful to have comments  like what 4 for nbFormat/nbformat_mino means